### DISCUSSION TITLE
Gazebo Ionic Tutorial Party Questions

### DISCUSSION THREAD

**Discussion by azeey** [gazebosim/gazebo_test_cases#1704] [General]
Thank you for participating in the Gazebo Ionic tutorial party! This discussion space is meant to be used for general questions regarding the tutorial party. If you have questions related to a particular tutorial, please leave your comment in the corresponding issue.

**Comment by Creator-1705** [comment]
where is the leader board link?

this link is not working : bit.ly/GzJettyLeaderboard

**Reply by azeey** [reply]
The main Readme has the link
